icu4c/source/common/restrace.h

```diff
@@ -0,0 +1,103 @@
+// © 2018 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+#ifndef __RESTRACE_H__
+#define __RESTRACE_H__
+
+#include "unicode/utypes.h"
+
+#if U_ENABLE_TRACING
+
+struct UResourceBundle;
+
+U_NAMESPACE_BEGIN
+
+class CharString;
+
+class U_COMMON_API ResourceTracer {
```

Comment:
Do you need U_COMMON_API here? Is it getting called from outside the common library?

Reply:
Yes, Windows builds complained.  I'll copy the boilerplate comment from somewhere else to clarify.